src/test/java/emissary/server/api/MetricsActionTest.java

```diff
@@ -1,16 +1,16 @@
 package emissary.server.api;
 
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.fail;
 
 import emissary.server.mvc.EndpointTestBase;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-public class MetricsActionTest extends EndpointTestBase {
+class MetricsActionTest extends EndpointTestBase {
```

Comment:
Should either delete this placeholder file or create a follow-on ticket to implement tests

Reply:
Ticket created